Core/Install/elxMacro.h

```diff
@@ -27,6 +27,7 @@
  * includes itkWin32Header.h.
  */
 #include "itkWin32Header.h"
+#include "itkMacro.h"
```

Comment:
@mstaring 

> should we also remove our redefinition of these macro's and include itkMAcro?

As far as I can see, this PR does not have a redefinition of `ITK_TEMPLATE_EXPORT` (anymore?), and neither does the current develop branch.

This PR does include itkMacro.